extra/write_throughput_benchmark.rst

```diff
@@ -0,0 +1,135 @@
+~~~~~~~~~~~~~~~~~~~~
+
+The easiest way to start a Citus Cluster is by vising the Citus Cloud dashboard. This dashboard allows you to choose different coordinator and worker node configurations and charges you by the hour. Once you picked your desired cluster setup, click on the "Create New Formation" button.
+
+.. image:: ../images/cloud-coordinator-worker-slider.png
+  :scale: 50 %
+  :align: center
+
+A pop-up will ask you the AWS region (US East, US West) where your formation will be created. Please remember the region where you created your Citus Cloud formation. We will use it in the next step.
+
+Please note that Citus Cloud automatically tunes your cluster based on your hardware configuration. If you're planning to run the following steps on your own cluster, you will need to manully increase :code:`max_connections = 303` on the coordinator and worker nodes.
+
+Create an Instance to Run pgbench
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pgbench is standard benchmarking tool provided by PostgreSQL. pgbench runs given SQL commands repeatedly and measures number of completed transactions per second.
+
+Since pgbench itself uses some CPU power to execute SQL commands, we recommend that you run pgbench on a separate machine than the one where your Citus cluster runs. This recommendation also follows pgbench's documentation.
+
+In these tests, we're going to create a separate EC2 instance to run pgbench, and place the instance in the same AWS region as our Citus cluster. We will also use a large EC2 (m4.16xlarge) instance to ensure pgbench itself doesn't become the performance bottleneck.
+
+Install pgbench
+~~~~~~~~~~~~~~~
+
+Once we create a new EC2 instance, we need to install pgbench on this instance.
+
+If you are running a **Debian** based system, simply type::
+
+  sudo apt-get update
+  sudo apt-get install postgresql-contrib-9.5
+
+If you are running a **RedHat** based system, simply type::
+
+  sudo yum update
+  sudo yum install postgresql95-contrib
+
+
+Benchmark INSERT Throughput
+---------------------------
+
+Initialize and Distribute Tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before we start, we need to tell pgbench to initialize the benchmarking environment by creating test tables. Then, we need to connect to the Citus coordinator node and distribute the table that we're going to run INSERT benchmarks on.
+
+To initialize the test environment and distribute the related table, you need to get a connection string to the cluster. You can get this connection string from your Citus Cloud dashboard. Then, you need to run the following commands::
+
+  pgbench -i connection_string_to_coordinator
+  psql connection_string_to_coordinator
+  
+  psql=> SELECT create_distributed_table('pgbench_history', 'aid');
+
+
+Create SQL File for pgbench
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pgbench runs the given SQL commands repeatedly and reports results. For this benchmark run, we will use the INSERT command that comes with pgbench.
```

Comment:
I kept this as-is since I wasn't sure if the reader would know what TPC-B is.

I mentioned that "the INSERT command that comes with pgbench". Happy to make this wording stronger.